app/controllers/Api/greetings_controller.rb

```diff
@@ -0,0 +1,10 @@
+module Api
```

Comment:
Kindly ensure that data is returned from the backend app.

![image](https://user-images.githubusercontent.com/27709832/228232209-3551b139-792c-4b6a-ba20-f90e546214ac.png)

![image](https://user-images.githubusercontent.com/27709832/228232716-10e0c64d-eb5f-4586-bd27-67897f8aa80e.png)

It currently returns a 404.